### PR TITLE
Docs-1638-CC-3.17.1

### PR DESCRIPTION
**Pull request by bmckercher123**
Product Version(s):

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

File: calico-cloud_versioned_docs/version-3.17/_includes/release-notes/_v3.17.0-release-notes.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+xx July 2023
```
Comment (Glen-Tigera) [inline calico-cloud_versioned_docs/version-3.17/_includes/release-notes/_v3.17.0-release-notes.mdx:1]:
```suggestion
xx August 2023
```

File: __tests__/crawler.test.js [hunk 2]
```diff
@@ -87,6 +87,7 @@ test("Crawl the docs and execute tests", async () => {
     `https://installer.calicocloud.io/manifests/v3.16.1-0/manifests`,
     `https://installer.calicocloud.io/manifests/v3.16.1-5/manifests`,
     `https://installer.calicocloud.io/manifests/v3.16.1-11/manifests`,
+    `https://installer.calicocloud.io/manifests/v3.17.1-11/manifests`,
```
Comment (Glen-Tigera) [inline __tests__/crawler.test.js]: @tmjd is this the right manifest version for v3.17?

Comment (tmjd) [reply __tests__/crawler.test.js]: This needs to be `v3.17.1-0`

File: calico-cloud_versioned_docs/version-3.17/variables.js [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
+  filesUrl_CE: 'https://downloads.tigera.io/ee/v3.17.1',
   tutorialFilesURL: 'https://docs.tigera.io/files',
   prodnameWindows: 'Calico Enterprise for Windows',
   rootDirWindows: 'C:\\TigeraCalico',
   nodecontainer: 'cnx-node',
   noderunning: 'calico-node',
-  cloudversion: 'v3.16.1-5',
-  clouddownloadurl: 'https://installer.calicocloud.io/manifests/v3.16.1-5',
+  cloudversion: 'v3.17.1-5',
+  clouddownloadurl: 'https://installer.calicocloud.io/manifests/v3.17.1-5',
```
Comment (tmjd) [inline calico-cloud_versioned_docs/version-3.17/variables.js]: These need to be updated to v3.17.1-0

File: calico-cloud_versioned_docs/version-3.17/variables.js [hunk 4]
```diff
@@ -1,22 +1,22 @@
...
   clouddownloadbase: 'https://installer.calicocloud.io',
   cloudoperatorimage: 'us-docker.pkg.dev/tigera-prod-01/calico-cloud-public/cc-operator',
-  imageassuranceversion: 'v1.5.6',
+  imageassuranceversion: 'v1.7.3',
```
Comment (tmjd) [inline calico-cloud_versioned_docs/version-3.17/variables.js]: Where did you get this number from? I think it should be v1.8.4

File: static/releases.html [hunk 5]
```diff
@@ -29,6 +29,13 @@
       
 
       <div style="font-weight: bold; margin-top: 4px">Calico Enterprise</div>
+      <div style="padding-left: 12px">
+        <a
+          href="/calico-enterprise/3.17/about/"
+          style="font-weight: normal; color: #333333"
+        >3.17</a
+        >
+      </div>
```
Comment (tmjd) [inline static/releases.html]: Is this related to CC?

File: calico-enterprise_versioned_docs/version-3.17/_includes/release-notes/_v3.17.1-release-notes.mdx [hunk 6]
```diff
@@ -0,0 +1,32 @@
+xx July 2023
```
Comment (tmjd) [inline calico-enterprise_versioned_docs/version-3.17/_includes/release-notes/_v3.17.1-release-notes.mdx]: I assume once you rebase this PR on latest master I expect this change will go away.

File: calico-cloud_versioned_docs/version-3.17/release-notes/index.mdx [hunk 7]
```diff
@@ -4,6 +4,44 @@ description: What's new, and why features provide value for upgrading.
 
 # Release notes
 
+## August 8, 2023
+
+### New and improved
```
Comment (tmjd) [inline calico-cloud_versioned_docs/version-3.17/release-notes/index.mdx:9]: This looks like the release notes have been copied from Enterprise. I don't think that is what has been done before when CC has rolled out a new version of Enterprise. Did you get this from PM? I think the release notes for CC are suppose to come from PM.

File: calico-cloud_versioned_docs/version-3.17/release-notes/index.mdx [hunk 8]
```diff
@@ -4,6 +4,44 @@ description: What's new, and why features provide value for upgrading.
 
 # Release notes
 
+## August 8, 2023
```
Comment (tmjd) [inline calico-cloud_versioned_docs/version-3.17/release-notes/index.mdx]: This needs to be updated to August 14, 2023

File: calico-cloud_versioned_docs/version-3.17/_includes/release-notes/_v3.17.0-release-notes.mdx (hunk 1 above)
Comment (tmjd) [reply calico-cloud_versioned_docs/version-3.17/_includes/release-notes/_v3.17.0-release-notes.mdx:1]: Does CC need this file?

File: __tests__/crawler.test.js (hunk 2 above)
Comment (bmckercher123) [reply __tests__/crawler.test.js]: Fixed.

File: calico-cloud_versioned_docs/version-3.17/_includes/release-notes/_v3.17.0-release-notes.mdx (hunk 1 above)
Comment (bmckercher123) [reply calico-cloud_versioned_docs/version-3.17/_includes/release-notes/_v3.17.0-release-notes.mdx:1]: This RN is not published and is just for archives. We only publish index.md

File: calico-cloud_versioned_docs/version-3.17/variables.js (hunk 3 above)
Comment (bmckercher123) [reply calico-cloud_versioned_docs/version-3.17/variables.js]: Fixed.

File: calico-cloud_versioned_docs/version-3.17/variables.js (hunk 4 above)
Comment (bmckercher123) [reply calico-cloud_versioned_docs/version-3.17/variables.js]: Updated.

File: calico-cloud_versioned_docs/version-3.17/release-notes/index.mdx (hunk 7 above)
Comment (bmckercher123) [reply calico-cloud_versioned_docs/version-3.17/release-notes/index.mdx:9]: Yes, I did copy from CE RN. PM can use this as starting point and at least use working links.

File: calico-cloud_versioned_docs/version-3.17/release-notes/index.mdx (hunk 8 above)
Comment (bmckercher123) [reply calico-cloud_versioned_docs/version-3.17/release-notes/index.mdx]: Fixed.

File: static/releases.html (hunk 5 above)
Comment (bmckercher123) [reply static/releases.html]: No. Deleting this.

File: calico-cloud_versioned_docs/version-3.17/_includes/release-notes/_v3.17.0-release-notes.mdx [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+    --set tags.tigera-prometheus-operator=false \
+    --set tigera-prometheus-operator.enabled=false
+```
+Or customize the Helm chart using [existing installation steps](getting-started/install-on-clusters/kubernetes/helm.mdx).
+
+### Bug fixes
+TBD
+
+### Known issues
+- Switching a cluster with Openshift v4.13 from iptables to eBPF causes the node to crash.
```
Comment (malavikabala) [inline calico-cloud_versioned_docs/version-3.17/_includes/release-notes/_v3.17.0-release-notes.mdx:34]: We should remove the notes about Openshift issues

File: calico-cloud_versioned_docs/version-3.17/_includes/release-notes/_v3.17.0-release-notes.mdx [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+- Upgrading to {{prodname}} 3.17.1 on Rancher/RKE from {{prodname}} 3.13.0 currently requires manually terminating the calico-node container for an upgrade to proceed.
```
Comment (malavikabala) [inline calico-cloud_versioned_docs/version-3.17/_includes/release-notes/_v3.17.0-release-notes.mdx:35]: Same here, we should remove this issue since it relates to RKE

File: calico-cloud_versioned_docs/version-3.17/release-notes/index.mdx (hunk 7 above)
Comment (malavikabala) [reply calico-cloud_versioned_docs/version-3.17/release-notes/index.mdx:9]: I think the releases notes in this case probably should be the same with a few minor updates

File: calico-cloud_versioned_docs/version-3.17/_includes/release-notes/_v3.17.0-release-notes.mdx [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+
+#### Support for DNS rules in clusters using NodeLocal DNSCache
+{{prodname}} has added support for DNS rules in clusters using NodeLocal DNSCache. Also related, there is new documentation on using Calico policy to secure DNS traffic within the cluster with NodeLocal DNSCache enabled. [Use NodeLocal DNSCache in your cluster](networking/configuring/node-local-dns-cache.mdx).
+
+#### Improved UI for configuring Workload-based Web Application Firewall (WAF)
+{{prodname}} includes updates to the UI that allows you to select which services are enabled for the Workload-based Web Application Firewall. [Web application firewall](threat/web-application-firewall.mdx).
+
+#### Wireguard support for AKS and EKS with Calico CNI
+{{prodname}} now offers official support for Wireguard when using Microsoft AKS or Amazon EKS with Calico CNI. This mode of deployment offers performance benefits and a more efficient routing table compared to using cloud provider CNIs.  [Encrypt data in transit](compliance/encrypt-cluster-pod-traffic.mdx).
+
```
Comment (malavikabala) [inline calico-cloud_versioned_docs/version-3.17/_includes/release-notes/_v3.17.0-release-notes.mdx:19]: Add release notes on RBAC updates:
Additional custom roles for Calico Cloud users
Users can now create custom role-based access controls for two new roles, "Usage Metrics" and "Image Assurance Admin"

**Review by malavikabala — COMMENTED** [review]
I've added a few comments - let me know if you need any other info from me. Sorry for the late review.

File: calico-cloud_versioned_docs/version-3.17/releases.json [hunk 12]
```diff
@@ -1,259 +1,263 @@
...
+    "title": "v3.17.1",
     "tigera-operator": {
       "image": "tigera/operator",
-      "version": "master",
+      "version": "v1.30.3",
       "registry": "quay.io"
     },
     "calico": {
-      "minor_version": "v3.24",
+      "minor_version": "v3.25",
```
Comment (Glen-Tigera) [inline calico-cloud_versioned_docs/version-3.17/releases.json:10]: I think this should be calico v3.26, this is what Enterprise v3.17.1 is based on

File: calico-enterprise_versioned_docs/version-3.17/variables.js [hunk 13]
```diff
@@ -5,7 +5,11 @@ const variables = {
   prodname: 'Calico Enterprise',
   prodnamedash: 'calico-enterprise',
   version: 'v3.17',
+<<<<<<< HEAD
   baseUrl: '/calico-enterprise/latest',
+=======
+  baseUrl: '/calico-enterprise/3.17',
+>>>>>>> ca381156 (Update docs for GA.)
```
Comment (Glen-Tigera) [inline calico-enterprise_versioned_docs/version-3.17/variables.js:12]: Conflict here

File: calico-cloud_versioned_docs/version-3.17/_includes/release-notes/_v3.17.0-release-notes.mdx (hunk 9 above)
Comment (bmckercher123) [reply calico-cloud_versioned_docs/version-3.17/_includes/release-notes/_v3.17.0-release-notes.mdx:34]: Deleted.

File: calico-cloud_versioned_docs/version-3.17/_includes/release-notes/_v3.17.0-release-notes.mdx (hunk 10 above)
Comment (bmckercher123) [reply calico-cloud_versioned_docs/version-3.17/_includes/release-notes/_v3.17.0-release-notes.mdx:35]: Deleted.

File: calico-cloud_versioned_docs/version-3.17/_includes/release-notes/_v3.17.0-release-notes.mdx (hunk 11 above)
Comment (bmckercher123) [reply calico-cloud_versioned_docs/version-3.17/_includes/release-notes/_v3.17.0-release-notes.mdx:19]: Added.

File: calico-cloud_versioned_docs/version-3.17/releases.json (hunk 12 above)
Comment (bmckercher123) [reply calico-cloud_versioned_docs/version-3.17/releases.json:10]: CE 3.17.1 went out with v3.25. @rene-dekker @danudey did we miss this update for CE?

**Comment by bmckercher123** [comment]
Closing out this PR in favor of this one: https://github.com/tigera/docs/pull/862.

File: calico-enterprise_versioned_docs/version-3.17/variables.js (hunk 13 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.17/variables.js:12]: I'm creating new PR to ensure I have accurate base.
